client/my-sites/plan-features/index.jsx

```diff
@@ -468,6 +483,7 @@ export default connect(
 				const newPlan = isNew( plan ) && ! isPaid;
 				const currentPlan = sitePlan && sitePlan.product_slug;
 				const showMonthlyPrice = ! relatedMonthlyPlan && showMonthly;
+				const currentPlanMatch = !! ( sitePlan && ( getPlanClass( sitePlan.product_slug ) === getPlanClass( plan ) ) );
```

Comment:
I've played with some implementations of selectors for matching plans. I think it could either be at the selector level or we could add a function to `lib/plans`.

`getPlanClass` works (stolen from https://github.com/Automattic/wp-calypso/blob/7e26047cb555453a3c6b3f7c77b8de73f0fe8e65/client/my-sites/plan-features/header.jsx#L158-L170), but feels a bit dirty. It seems to be intended for CSS classes.